code/datums/actions/action.dm

```diff
@@ -1,3 +1,14 @@
+/// Actions that toggle on click/trigger
+#define ACTION_TOGGLE "toggle"
+/// Actions that trigger when clicked/triggered
+#define ACTION_CLICK "click"
+/// Actions that get selected and can be targeted when clicked/triggered
+#define ACTION_SELECT "select"
+
+#define KEYBINDING_NORMAL "normal_trigger"
+#define KEYBINDING_ALTERNATE "alternate_trigger"
```

Comment:
these need to be in the defines file
side note autodocing normal and alternate would be nice